website/docs/d/efs_access_point.html.markdown

```diff
@@ -0,0 +1,45 @@
+---
+subcategory: "EFS"
+layout: "aws"
+page_title: "AWS: aws_efs_access_point"
+description: |-
+  Provides an Elastic File System (EFS) Access Point data source.
+---
+
+# Data Source: aws_efs_access_point
+
+Provides information about an Elastic File System (EFS) Access Point.
+
+## Example Usage
+
+```hcl
+resource "aws_efs_file_system" "test" {
+  creation_token = "some-token"
+}
+
+resource "aws_efs_access_point" "test" {
+  file_system_id = "${aws_efs_file_system.test.id}"
+}
+
+data "aws_efs_access_point" "test" {
+  access_point_id = "${aws_efs_access_point.test.id}"
+}
```

Comment:
We should avoid showing data sources dependent on their associated resources in data source examples because this is not a pattern recommended in Terraform.

```suggestion
data "aws_efs_access_point" "test" {
  access_point_id = "fsap-12345678"
}
```